examples/hotrod/kubernetes/base/hotrod/deployment.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: example-hotrod
+  name: example-hotrod
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: example-hotrod
+  strategy: {}
+  template:
+    metadata:
+      labels:
+        app: example-hotrod
+    spec:
+      containers:
+      - image: jaegertracing/example-hotrod:latest
+        name: example-hotrod
+        args: ["all"]
+        env:
+          - name: JAEGER_AGENT_HOST
+            value: jaeger.example-hotrod.svc.cluster.local
+          - name: JAEGER_AGENT_PORT
+            value: "6831"
+        ports:
+          - containerPort: 8080
+            name: frontend
+          - containerPort: 8081
+            name: customer
+          - containerPort: 8083
+            name: route
+        resources:
+          limits:
+            cpu: 100m
+            memory: 100M
+          requests:
+            cpu: 100m
+            memory: 100M
+      - image: redis:latest
```

Comment:
why is this needed?

Reply:
The `hotrod` app appears to need `redis`. When I was viewing the traces it was failing to connect.

Reply:
Hotrod binary has no external dependencies, it fakes them.

Reply:
Gotcha. I'll remove redis, then.